src/tools/Tool.ts

```diff
@@ -6,11 +6,11 @@ export type RenderCanvas = ( canvasElement: CanvasElement, layer: Konva.Layer) =
 
 export interface Tool {
   name: string;
-  colour: string;
-  size: number;
+  colour?: string;
+  size?: number;
```

Comment:
Made these parameters optional because not all tools are in need of a color and size.